temba_client/tests.py

```diff
@@ -70,11 +70,15 @@ def utcoffset(self, dt):
 
     def test_format_iso8601(self):
         d = datetime.datetime(2014, 1, 2, 3, 4, 5, 6, UtilsTest.TestTZ())
-        self.assertEqual(format_iso8601(d), '2014-01-02T08:04:05.000006')
+        self.assertEqual(format_iso8601(d), '2014-01-02T08:04:05.000006Z')
 
     def test_format_iso8601_should_return_none_when_no_datetime_given(self):
```

Comment:
Include this test case in test method above - stylistically we tend to do one test method per method being tested